frontend/src/components/dashboard/settings/custom-from-address/CustomFromAddress.module.scss

```diff
@@ -28,15 +29,12 @@
     }
 
     .status {
-      @include button(white);
-      border: 1px solid $primary;
-      color: $primary;
-      &:hover {
-        background: $neutral-light;
-      }
+      @extend %outline-btn;
     }
     .request {
-      @include button($secondary) @include mobile() {
+      @extend %secondary-btn;
```

Comment:
How do we decide when to use `@include button(...) `vs `@extend`

Reply:
Mixins duplicate styles while extend doesn't. Generally if we just want to reuse static styles, we should try to use `@extend` instead of `@include` to reduce the size of the generated CSS.

**Using mixins:** 
```sass
@mixin button($background-color) {
    background-color: $background-color
}

.button-mixin-1 {
    @include button(red)
}

.button-mixin-2 {
    @include button(red)
}
```

Compiles to:
```css
.button-mixin-1 {
  background-color: red;
}

.button-mixin-2 {
  background-color: red;
}
```

**Using extends:**
```sass
%button {
    background-color: red
}

.button-extend-1 {
    @extend %button
}

.button-extend-2 {
    @extend %button
}
```

Compiles to:
```css
.button-extend-2, .button-extend-1 {
  background-color: red;
}
```

**Reference**
- [When to use extends vs mixins](https://medium.com/stories-from-the-keen/when-to-use-extends-vs-mixins-in-sass-b09d55abd53)
- https://www.sassmeister.com/ to test out

Reply:
I see.  Ok, good to know! Thanks

```scss
@mixin button($background-color) {
    background-color: $background-color
}

%secondary-btn {
    @include button(#FFF);
}

.ext-1 {
    @extend %secondary-btn
}

.ext-2 {
    @extend %secondary-btn
}

.inc-1 {
    @include button(#FFF);
}

.inc-2 {
    @include button(#FFF);
}
```
compiles to
```css
.ext-2, .ext-1 {
  background-color: #FFF;
}

.inc-1 {
  background-color: #FFF;
}

.inc-2 {
  background-color: #FFF;
}
```